client/state/billing-transactions/individual-transactions/actions.js

```diff
@@ -0,0 +1,42 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import {
+	BILLING_TRANSACTION_REQUEST,
+	BILLING_TRANSACTION_REQUEST_FAILURE,
+	BILLING_TRANSACTION_REQUEST_SUCCESS,
+} from 'state/action-types';
+import wp from 'lib/wp';
+
+export const requestBillingTransaction = transactionId => dispatch => {
+	dispatch( {
+		type: BILLING_TRANSACTION_REQUEST,
+		transactionId,
+	} );
+
+	return wp
+		.undocumented()
+		.me()
+		.getReceipt( transactionId, { format: 'display' } )
+		.then( receipt => {
+			dispatch( {
+				type: BILLING_TRANSACTION_REQUEST_SUCCESS,
```

Comment:
For billing transactions we have a separate `BILLING_TRANSACTIONS_REQUEST_SUCCESS` for managing request status, and `BILLING_TRANSACTIONS_RECEIVE` to actually receive transactions, which usually happens on a successful request. I wonder if we want to be consistent with that.